crates/tabby-scheduler/src/crawl/mod.rs

```diff
@@ -35,6 +41,12 @@ async fn crawl_url(start_url: &str) -> anyhow::Result<impl Stream<Item = KatanaR
         }
     });
 
+    tokio::spawn(async move {
+        while let Ok(Some(line)) = stderr.next_line().await {
+            on_stderr_line(line);
```

Comment:
just invoke logkit::info here?

Reply:
Got it, wasn't aware this would work across both crates

Reply:
Can't we do the same for `on_process_url` too then?

Reply:
I commented on this in last round, guess it's missed?
https://github.com/TabbyML/tabby/pull/2519#discussion_r1657991256